dashboard/src/main/resources/templates/dashboard.ftl

```diff
@@ -10,5 +10,7 @@
         <li><a href='${artifact?replace(":", "_")}.html'>${artifact}</a></li>
       </#list>
       </ul>
+      foo
+      <p id='updated'>Last generated at ${lastUpdated}</p>
```

Comment:
So this is essentially when the page was generated?

Reply:
Correct. I mostly threw it in because otherwise a failed run generates no obvious output here. This will tell us if it's stale.